docs/02_getting_started.md

```diff
@@ -37,10 +37,10 @@ You may even find some new, indispensable tools to help with continuous integrat
 
 **Git is:**
 
-- a distributed version control system or DVCS.
-- free and open source.
-- designed to handle everything from small to very large projects with speed and efficiency.
-- easy to learn and has a tiny footprint with lightning fast performance.
+- A distributed version control system (DVCS)
+- Free and open source
+- Designed to handle everything from small to very large projects with speed and efficiency
+- Easy to learn and has a tiny footprint with lightning fast performance
```

Comment:
```suggestion
- a distributed version control system (DVCS).
- free and open source.
- designed to handle everything from small to very large projects with speed and efficiency.
- easy to learn and has a tiny footprint with lightning fast performance.
```

These are lowercase because they are finishing the sentence, `Git is ...` and ended with a period.

Reply:
I changed this based on [what the GitHub Brand Guide says about lists](https://brand.github.com/content/grammar#lists), but this is a fair point. Thanks!